flax/linen/sequential.py

```diff
@@ -0,0 +1,13 @@
+from typing import Sequence
+from flax.linen.module import Module
+from typing import Type
+
+
+
+class Sequential(Module):
+  layers: Sequence[Type[Module]]
+
+  def __call__(self, x):
+    for layer in self.layers:
+      x = layer(x)
```

Comment:
Here you could perhaps first do:
```
if not callable(layer):
  raise ValueError(f"Each layer should implement `__call__`, but layer `{layer}` is not callable.")
```
This should make pytype stop complaining when you declare `layers: Sequence[Module]`.